docs/source/reference/ext.rst

```diff
@@ -19,13 +19,15 @@ CuPy-specific functions are placed under ``cupyx`` namespace.
 DLPack utilities
 ----------------
 
-This is a helper function for creating a :class:`cupy.ndarray` from a DLPack tensor.
+Below are helper functions for creating a :class:`cupy.ndarray` from either a DLPack tensor
+or any object supporting the DLPack data exchange protocol.
 For further detail see :ref:`dlpack`.
 
 .. autosummary::
    :toctree: generated/
 
    cupy.fromDlpack
```

Comment:
Could you remove this line from the list?

Reply:
Thanks! Done in 918f3f7.